stories/Constants.ts

```diff
@@ -0,0 +1,22 @@
+import {useGlobals} from "@storybook/addons";
+
+export const COMPONENTS_TITLE = 'Components';
+
+export const createStoryTitle = (componentName: string) => {
```

Comment:
fairly general name for a specific action

Reply:
Just wanted to abstract it away incase we need to change in the future is easy (looks a bit overkill for now I agree but worth it in long run lol)